index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./index.css" />
+    <title>Google Clone</title>
+  </head>
+  <body>
+    <section class="page">
+      <img
+        class="logo"
+        src="https://www.google.it/images/branding/googlelogo/1x/googlelogo_color_272x92dp.png"
+      />
+      <br /><input
+        id="searchme"
+        class="search"
+        title="Search"
+        type="text"
+      /><br />
+      <div class="Bcontainer">
+        <div class="button left" id="btn-left">Google Search</div>
+        <div class="button right">I'm Feeling Lucky</div>
+      </div>
+    </section>
+  </body>
+</html>
```

Comment:
Should have linked the javascript file on the html. 

There is a forward slash on line 21.

Can you use div for a button on html?